contracts/SportMarkets/SportPositions/SportPositionalMarket.sol

```diff
@@ -206,6 +187,23 @@ contract SportPositionalMarket is OwnedWithInit, ISportPositionalMarket {
         return (gameDetails.gameId, gameDetails.gameLabel);
     }
 
+    function getParentMarketPositionsUint() public view override returns (uint position1, uint position2) {
```

Comment:
This wont work for already created markets, how do we handle that during release?